tests/testsuite/build.rs

```diff
@@ -4373,3 +4373,65 @@ fn target_filters_workspace_not_found() {
         .with_stderr("[ERROR] no library targets found in packages: a, b")
         .run();
 }
+
+#[cfg(unix)]
```

Comment:
Despite this being unix only I personally like having tests like this, so I'd definitely keep it

Reply:
Doesn't this cause an abnormal exit on all systems?

Reply:
Yes, I just didn't want to deal with figuring out the text of the error message on every platform.  Windows is something like "3221226505", but is it like that on all versions of windows? (probably?)  What about non-tier-1 platforms?  I'm already concerned this test might be flaky on weird platforms.

Reply:
For Windows error codes are actually best rendered as hex -- 0xC0000409 instead of 3221226505 -- and I've meant to do that in libstd for some time now...